### ISSUE TITLE
Do a manual check of the mapping between introductions and individual MP

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#175]
We should take a random sample size of roughly 300 introductions (roughly 75 pages per person and 3 pages per decade) and then manually check if the person mapping is ok.

@rbbby will start to set up a CSV file with one intro per row and with relevant info to do check if the mapping is correct? If the mapping is unknown, give the Q-number from wikidata. Try to find.

@rbbby start out by testing this

**Comment by rbbby** [comment]
Annotated 100+ observations for 0.4.4 where everything looks good.
After implementing BERT models for 0.4.5 I annotated 300 new observations and also created a gold standard for identifying speech holders name in intros (input/accuracy/intro_parser_gold_standard.csv).
Everything looks good but the need for some kind of dependency parsing of intros (which we already know). I added a pseudo dependency parser by expanding our regex pattern which works better (but not still perfectly).
